llmfoundry/tokenizers/tiktoken.py

```diff
@@ -26,7 +26,7 @@ def __init__(self,
                  eos_token: Optional[str] = '<|endoftext|>',
                  bos_token: Optional[str] = '<|endoftext|>',
                  pad_token: Optional[str] = None,
-                 **kwargs: Dict[str, Any]):
+                 **kwargs: Any):
```

Comment:
Aren't `**kwargs` always a dict of str to Any??

Reply:
You're supposed to type **kwargs withthe type of the values only

Reply:
TIL. Thanks!